articles/cognitive-services/Anomaly-Detector/How-to/deploy-anomaly-detection-on-iot-edge.md

```diff
@@ -53,7 +53,7 @@ Learn how to deploy the Cognitive Services [Anomaly Detector](../anomaly-detecto
 
 6. Select **Update**
 
-7. Select **Next: Routes** to define your route. You define all messages from all modules to go to Azure IoT Hub.
+7. Select **Next: Routes** to define your route. You define all messages from all modules to go to Azure IoT Hub. Please refer to [Establish routes in IoT Edge](https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2020-11) on how to declare the desired route.
```

Comment:
the presence of en-us will hardcode this link to a single locale. When omitted the link becomes locale agnostic which is important when this article gets localized into other languages.

```suggestion
7. Select **Next: Routes** to define your route. You define all messages from all modules to go to Azure IoT Hub. Please refer to [Establish routes in IoT Edge](https://docs.microsoft.com/azure/iot-edge/module-composition?view=iotedge-2020-11) on how to declare the desired route.
```